src/testUtils.tsx

```diff
@@ -3,16 +3,14 @@ import { cleanup, render } from '@testing-library/react'
 import { afterEach } from 'vitest'
 import { Provider } from 'react-redux'
 
-import store from '@/store'
-
 afterEach(() => {
   cleanup()
 })
 
 // Provide Redux store for renders
 const customRender = (ui: React.ReactElement, options = {}) =>
   render(ui, {
-    wrapper: ({ children }) => <Provider store={store}>{children}</Provider>,
+    wrapper: ({ children }) => <Provider>{children}</Provider>,
```

Comment:
**Suggestion:** Provide the Redux store to the Provider component. It currently doesn't receive any store.
```suggestion
    wrapper: ({ children }) => <Provider store={store}>{children}</Provider>,
```